### DISCUSSION TITLE
Accessibility considerations for multiple inputs

### DISCUSSION THREAD

**Discussion by Khushli** [twilio-labs/paste#2178] [General] [answered]
Hi, we have a usecase where user can add multiple input values for the same field. Can the additional input fields work without labels? I have shown only 3 inputs in the example but a user can add up to 10 inputs. 

<img width="665" alt="Screen Shot 2022-02-02 at 2 32 55 PM" src="https://user-images.githubusercontent.com/87436655/152223757-cb2198cd-aec4-4d30-bda2-1ada09da761f.png">

**Comment by gloriliale** [comment, marked as answer]
Hi @Khushli! I've discussed your design w/ the team, and the simplest way forward that ensures accessibility for these inputs would be to note that they should be built inside a fieldset and that each input should have a unique hidden aria-label. It can be achieved by wrapping the inputs with `<Box as="fieldset">`, and labeling this fieldset with `<Label as="legend">`

Let me know if you have any questions!

**Reply by gloriliale** [reply]
Here's a[ codesandbox example ](https://codesandbox.io/s/multiple-inputs-w-aria-label-7jl10?file=/src/index.tsx)of what this would look like as well, @Khushli!
